src/js/multiDomainFirstParties.js

```diff
@@ -182,7 +169,7 @@ var _multiDomainFirstPartiesArray = [
   ["walmart.com", "wal.co"],
   ["wellsfargo.com", "wf.com"],
   ["wikia.com", "wikia.net", "nocookie.net"],
-  ["wikipedia.org", "wikimedia.org", "wikimediafoundation.org", "wiktionary.org",
+  ["wmflabs.org","wikipedia.org", "wikimedia.org", "wikimediafoundation.org", "wiktionary.org",
```

Comment:
Nit: missing a separating space.

Could we move it to the back of Wikipedia entries somewhere? We should have the most recognizable sites up front probably.